src/components/dapp-staking/dapp/ProjectDetails.vue

```diff
@@ -113,52 +125,55 @@
                 <span class="text--tooltip">{{ $t('common.twitter') }}</span>
               </q-tooltip>
             </button>
+            <button v-if="it.type === CommunityType.Reddit" class="box--share btn--primary">
+              <div class="icon--social">
+                <a :href="it.handle" target="_blank" rel="noopener noreferrer">
+                  <astar-icon-base :icon-name="CommunityType.Reddit">
+                    <astar-icon-reddit />
+                  </astar-icon-base>
+                </a>
+              </div>
+              <q-tooltip>
+                <span class="text--tooltip">{{ $t('common.reddit') }}</span>
+              </q-tooltip>
+            </button>
+            <button class="box--share btn--primary">
```

Comment:
There are 2 facebook icons. Perhaps add `v-if="it.type === CommunityType.Facebook"`?
![image](https://user-images.githubusercontent.com/92044428/201336929-a78770bc-7148-4b90-a228-a47340886d6a.png)

Reply:
yes my mistakes, here's [fix](https://github.com/AstarNetwork/astar-apps/pull/575/commits/057fe236bc863d486a278f3032791956c2da2773)